### DISCUSSION TITLE
How  to get my deleted repository back?

### DISCUSSION THREAD

**Discussion by NishantTomer5601** [community/community#48290] [Repositories] [answered]
### Select Topic Area

Product Feedback

### Body

Can anybody help me in getting my repo back. I would be very thankful for it.

**Comment by MotumaAdmasu** [comment]
In the upper-right corner of any page, click your profile photo, then click Settings. In the "Code planning, and automation" section of the sidebar, click Repositories. Under "Repositories", click Deleted repositories. Next to the repository you want to restore, click Restore.

or may be you can find it  on your local repo list if you didnt pull it to the main yet.

**Comment by NishantTomer5601** [comment]
Sir, I am unable to restore the deleted repo.
What should I do?

**Reply by MotumaAdmasu** [reply]
did you create the repo through github web or github desktop?
if it is through web and you deleted it, the chance of you getting it back is very low

**Comment by NishantTomer5601** [comment]
I did it through gitHub website.

**Comment by MotumaAdmasu** [comment]
it depends when you deleted the repo and if the repo was forked or not.
Github documentation says it will show up in deleted repo within the hour of you deleted it, but i dont it will

**Comment by NishantTomer5601** [comment]
it was not forked, any chance to get it back now?

**Comment by MotumaAdmasu** [comment]
did you already checked if it was in this link?

https://github.com/settings/deleted_repositories

it will probably in there by now if it wasnt forked.
like this below
![image](https://user-images.githubusercontent.com/17107139/220691360-c94a9cf8-25ef-40c1-9c2a-b3c6f37842bd.png)

**Reply by MotumaAdmasu** [reply]
can you please mark my comment as an answer if it helped you?

**Comment by NishantTomer5601** [comment]
sure.

**Reply by MotumaAdmasu** [reply]
your discussion still says Unanswered. can you please mark it as answered?

**Comment by NishantTomer5601** [comment]
It is being shown in this list of deleted repos. Sent from Mail for Windows From: Motuma AdmasuSent: 22 February 2023 21:54To: community/communityCc: Nishant Tomer; MentionSubject: Re: [community/community] How to get my deleted repository back? (Discussion #48290) did you already checked if it was in this link?https://github.com/settings/deleted_repositories—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Reply by MotumaAdmasu** [reply]
Did you tried to restore it then?
if it is shown in deleted repositories

**Comment by NishantTomer5601** [comment]
yes , I did.
Can you tell me if I can get my working days being marked green  as they
were earlier on the profile page before deleting the repo.

**Reply by MotumaAdmasu** [reply]
and please don't forget to mark my answer.(the one with the picture). it will really helps my profile. thanks.

**Comment by MotumaAdmasu** [comment, marked as answer]
if you are asking about commit activities on the repo, i don't think there is anything you can do to get that back. but may be it will show up by itself.

**Comment by NishantTomer5601** [comment]
Done.

**Comment by Lopdee** [comment]
Hello I want to please reinstall my deleted repo alx_zero_day

**Reply by saifeddineelhanoune** [reply]
you can post it as a question to the related context and you'll get answers
first you should go to the settings>repositories>deleted-repositories
the repository deleted will no longer appear if 90 days have past

**Comment by ilmudinSolanki** [comment]
how can i get back all disgard changes which chages are not commited and not saved in any branch

**Reply by saifeddineelhanoune** [reply]
you can discard changes for all files
git checkout -- .
if you want to reset all changes including untracked files you can use the git reset --hard command
